### ISSUE TITLE
orm.DB interface is incomplete

### ISSUE THREAD

**Issue by heynemann** [go-pg/pg#327]
I'm trying to use orm.DB in my model methods to ensure that it gets used instead of an actual pg.DB because I sometimes call the operation in a transaction an sometimes in a regular db session.

The problem is it is missing some methods. The one I noticed is:
```
_, err := db.Model(&e).
		OnConflict("(id) DO UPDATE").
		Set("name = ?name").
		Create()
```
This fails with `type orm.DB has no field or method Model`.

**Comment by vmihailenco** [comment]
Fixed in v4.9.4. Thanks for the report.

**Comment by heynemann** [comment]
Thanks!!!
